content/vault/v1.20.x/content/api-docs/system/storage/raftautosnapshots.mdx

```diff
@@ -81,7 +81,13 @@ parameters in the context of AWS EKS & S3 configuration.
 
 - `aws_s3_region` `(string: <required>)` - AWS region bucket is in.
 
-- `aws_access_key_id` `(string)` - AWS access key ID.
+- `aws_access_key_id` `(string)` - AWS access key ID. Make sure that this account has the following permissions in S3, so that Vault can store and manage the snapshots:
```

Comment:
```suggestion
- `aws_access_key_id` `(string)` - AWS access key ID. The assocaited AWS account
  must have the following permissions in S3 so that Vault can store and manage
  the snapshots:
```